src/components/PlaidLink/index.native.js

```diff
@@ -23,6 +24,10 @@ function PlaidLink(props) {
             },
         });
 
+        return () => {
+            dismissLink();
```

Comment:
I see this is beyond the proposal. What does this fix?

Reply:
<img width="893" alt="Screenshot 2023-10-19 at 9 19 15 PM" src="https://github.com/Expensify/App/assets/16595846/2e695b05-f5c2-4c48-877c-f57c0bf2bb5e">
Already there in my proposal.
when a component is destroyed we need to dismiss the link  
when onexit event trigger plaid component will destroy based on this `subStep`
https://github.com/Expensify/App/blob/c77814fd5e643cba78e8c8294370d78f633a59c5/src/pages/ReimbursementAccount/BankAccountPlaidStep.js#L105

Reply:
Ah ok, but what does this fix exactly?
What issue do you have currently in production app?

Reply:
when the component is destroyed if the plaid popup opens means it will close the popup.
in the current issue   was  popup is opened repeatedly after resuming the app

Reply:
ok so with this fix only, duplicated popup bug is replaced with repeatedly close & open popup 2 times.
So only 1 instance exists, which is a little better than original bug. But still we should fix this for better user experience which is the reason for introducing new Onyx key
Thanks for the explanation.

https://github.com/Expensify/App/assets/108292595/1002af31-38f7-4b01-be0c-907dbae452f7